modules/social_features/social_profile/social_profile.install

```diff
@@ -202,3 +204,28 @@ function social_profile_update_121001(): void {
 function social_profile_update_130001(): void {
   user_role_grant_permissions('sitemanager', ['create terms in profile_tag']);
 }
+
+/**
+ * Update views /all-members to hide blocked members.
+ */
+function social_profile_update_130002(): void {
```

Comment:
I picked 130002 so it runs, but I'm not so sure we want to want for 13.x to be released? Or do we?

Reply:
130002 is fine